src/main.py

```diff
@@ -38,7 +38,7 @@ def main_loop():
 
         except Exception as e:
             traceback.print_exc()
-            display_error()
+            display_error(["ERROR : ", "Trying again"])
```

Comment:
I think all caps is too aggressive for an error that the user is expected to see.
```suggestion
            display_error(["Error"])
```